docs/context/hypothetical-examples/handoff-v1.jsonld

```diff
@@ -0,0 +1,15 @@
+{
+    "@context":
+    {
+      "@version": 1.0,
+      "handoff": "https://project.factom.com/contexts/handoff-v1#",
+      "xsd": "http://www.w3.org/2001/XMLSchema#",
+      "schema": "http://schema.org/",
+      "@type": "schema:TransferAction",
+      "recipient": "schema:recipient",
```

Comment:
@hannemennah 
This handoff document looks good and Mavennet will also be using it for interoperability purposes. 
In order to adapt it for the pipeline use case, we would like to add more fields like

1. delivery volume
2. Route (which pipeline - the number or name)
3. Nomination month (a time when it happened)

Let us know what you think about it.